src/app/(social-page)/profile/[id]/layout.tsx

```diff
@@ -0,0 +1,302 @@
+/* eslint-disable @next/next/no-img-element */
+'use client'
+
+import '../../../globals.css'
+import React, { useState, useEffect, useContext, createContext } from 'react'
+import { nunito } from '../../../ui/fonts'
+import {
+  Button,
+  Avatar,
+  Tab,
+  Tabs,
+  TabsHeader,
+  Tooltip,
+} from '@material-tailwind/react'
+import Cookies from 'js-cookie'
+import { useForm } from 'react-hook-form'
+import { usePathname, useSearchParams, useRouter } from 'next/navigation'
+import { useDebouncedCallback } from 'use-debounce'
+import axios, { AxiosResponse } from 'axios'
+import { CheckCircleFill, XCircleFill, Camera } from 'react-bootstrap-icons'
+import Link from 'next/link'
+import toast, { Toaster } from 'react-hot-toast'
+import { PROFILE_TABS } from '../../../constant'
+import clsx from 'clsx'
+import Image from 'next/image'
+import ChangeAvatarUser from '@/app/ui/social-page/profile/ChangeAvatarUser'
+import ChangeCoverUser from '@/app/ui/social-page/profile/ChangeCoverUser'
+import CustomToaster from '@/app/ui/common/custom-toaster'
+import { JWT_COOKIE } from '@/app/constant'
+
+const FormContext = createContext(null)
+
+function AvatarAndCoverUser({ register, getValues, user, isProfileLoginUser }) {
+  const {
+    setInputs,
+    croppedAvatar,
+    setCroppedAvatar,
+    coverImage,
+    setCoverImage,
+  } = useContext(FormContext)
+
+  const [openChangeCover, setOpenChangeCover] = useState(false)
+  const handleOpenChangeCover = () => {
+    setOpenChangeCover((e) => !e)
+  }
+
+  const handleChangeAvatar = () => {
+    const formData = new FormData()
+
+    formData.append('avatar', getValues('avatar')[0]) // Ensure this is a File object
+
+    axios
+      .putForm(
+        `${process.env.NEXT_PUBLIC_SERVER_HOST}/user/profile/avatar`,
+        formData,
+        {
+          headers: {
+            Authorization: `Bearer ${Cookies.get(JWT_COOKIE)}`,
+            'Content-Type': 'multipart/form-data',
+          },
+        }
+      )
+      .then((response) => {
+        // Handle success response
+        toast.success('Avatar updated successfully')
+      })
+      .catch((error) => {
+        // Handle error response
+        toast.error('Failed to update avatar')
+        console.log(error)
+      })
+  }
+
+  const handleChangeCover = () => {
+    const formData = new FormData()
+    formData.append('cover', getValues('cover')[0]) // Ensure this is a File object
+
+    axios
+      .put(
+        `${process.env.NEXT_PUBLIC_SERVER_HOST}/user/profile/cover`,
+        formData,
+        {
+          headers: {
+            Authorization: `Bearer ${Cookies.get(JWT_COOKIE)}`,
+            'Content-Type': 'multipart/form-data',
+          },
+        }
+      )
+      .then((response) => {
+        toast.success('Cover updated successfully')
+      })
+      .catch((error) => {
+        toast.error('Failed to update cover')
+      })
```

Comment:
trong catch thì ko dùng cái này
```suggestion
      .catch((error) => {
        toast.error(error.response?.data?.error?.message || 'Lỗi không xác định')
      })
```
sửa lại những cái toast nào b đã làm như v